stacks-signer/src/signer.rs

```diff
@@ -979,10 +973,12 @@ impl Signer {
                     self.process_dkg(stacks_client, point, current_reward_cycle);
                 }
                 OperationResult::SignError(e) => {
+                    warn!("Signer #{}: Received a Sign error: {e:?}", self.signer_id);
                     self.process_sign_error(e);
                 }
                 OperationResult::DkgError(e) => {
                     warn!("Signer #{}: Received a DKG error: {e:?}", self.signer_id);
+                    // TODO: process these errors and track malicious signers to report
```

Comment:
Just want to confirm that there's a ticket for this?

Reply:
There is now. https://github.com/stacks-network/stacks-core/issues/4426

Not sure how to do malicious reports in DKG case. I think they can only really be done in Epoch 3.0 onwards as we include a bitvec in a block...Sign case also needs to be updated to use this bitvec, but we can at least emit a block rejection with a stated reason (including all the malicious signer ids)